ODIN_II/SRC/netlist_statistic.cpp

```diff
@@ -12,7 +12,7 @@
 #include "odin_util.h"
 #include "vtr_memory.h"
 
-#define UNUSED_NODE_TYPE -1
+#define UNUSED_NODE_TYPE 0
```

Comment:
Comment what this means, and why 0 is a good value (not sure why it used to be -1, and is now 0).

Reply:
The Odin-II netlist statistic function does not show nodes with this value in the output file. I modified this value also to avoid showing cell types that do not exist in the netlist. We previously had "Number of <MEMORY>: 0" or "Number of <ADD>: 0" I made this change to avoid showing these logs.

Reply:
Good to comment that that is what it is for then.  I think it would be worth adding a comment to that effect.